services/brig/src/Brig/Team/API.hs

```diff
@@ -110,10 +121,14 @@ teamSizePublic uid tid = do
 teamSize :: TeamId -> (Handler r) TeamSize
 teamSize t = lift $ TeamSize.teamSize t
 
-getInvitationCode :: TeamId -> InvitationId -> (Handler r) FoundInvitationCode
+getInvitationCode ::
+  (Member Store.InvitationCodeStore r) =>
+  TeamId ->
+  InvitationId ->
+  (Handler r) FoundInvitationCode
 getInvitationCode t r = do
-  code <- lift . wrapClient $ DB.lookupInvitationCode t r
-  maybe (throwStd $ errorToWai @'E.InvalidInvitationCode) (pure . FoundInvitationCode) code
+  inv <- lift . liftSem $ Store.lookupInvitation t r
```

Comment:
`lookupInvitationCode` changed its name. maybe i should go through all the changes in the implementation that you did during the move, and see if they are all semantics-preserving?

Reply:
I was careful to and Igor also had a look at it but more eyes can’t hurt. 

The reason why I did this is because lookupInvitation was two functions in a trench coat. splitting it up simplified things and separation of concerns.